communications/blogs/turing_joins_binder_federation.md

```diff
@@ -0,0 +1,102 @@
+It is a community of scientists and open-source developers dedicated to scientific reproducibility, communicating ideas through interactive computing, and providing real-time access to results and analyses.
+The Binder Federation provides interactive data analytics environments to roughly 10,000 unique repositories via 400,000 launches a month, and it relies on its team of partner BinderHubs to collaborate in providing this service.
+
+[GKE](https://jupyterhub-team-compass.readthedocs.io/en/latest/team.html#binder-team), [OVH](https://www.ovh.com/world/), [GESIS](https://notebooks.gesis.org/about/) and now new members the Turing pool their resources to create a sustainable, open-source infrastructure that allows easy creation, sharing of and access to reproducible computational environments.
+Pooling resources in such a way is an important step for the resilience and sustainability of Project Binder since the dependency between mybinder.org and its funding is now spread across many sources.
+It also facilitates the decentralisation of the knowledge required to deploy and manage a BinderHub.
+
+_Turing Way_ contributor [Dr Sarah Gibson](https://www.turing.ac.uk/people/researchers/sarah-gibson) is also a [mybinder.org](https://mybinder.org) operator.
+The operators are responsible for the day-to-day operations and maintainance of the main website.
+She was therefore perfectly positioned to lead this project and bring the Turing into the Binder Federation.
+
+> I'm so excited that we've successfully merged our resources at the Turing into the mybinder.org infrastructure!
+> This achievement marks Binder's maturity as a federated project and the Turing's commitment to supporting open-source tooling and communities.
+>
+> -- Dr Sarah Gibson
+
+![Federation](https://miro.medium.com/max/1666/1*KU35naJhl1LSDxKY8gog4g.png)
+
+## How Can You Get Involved?
+
+If this has piqued your interest, there are a few ways to get involved in the Binder Federation!
+
+One way is to become a [mybinder.org](https://mybinder.org) operator.
+This is a low-tech contribution which involves hanging out in the project's [gitter room](https://gitter.im/jupyterhub/binder) answering questions, providing constructive input in the [GitHub repo](https://github.com/jupyterhub/binderhub), and being an active member of the community.
+Invitations to join the operating team are extended to outstanding members of our community.
+
+Alternatively, you could join the federation by facilitating the deployment of a  BinderHub!
+You can donate a cluster or cloud credits to the mybinder.org operating team.
+Your deployment would then be managed from the [mybinder.org-deploy repo](https://github.com/jupyterhub/mybinder.org-deploy) with input from the team and community.
+This is a more high-tech contribution but over a shorter time period.
+
+Lastly rather than joining the federation, you could deploy your own BinderHub for your organisation.
```

Comment:
```suggestion
Lastly, rather than joining the Federation, you could deploy your own BinderHub for your organisation.
```

Reply:
I think there are a few more instances of "federation" rather than "Federation" waiting in other comments - something to check after finishing the first round of reviews.